src/components/v1/ProjectPayoutMods.tsx

```diff
@@ -437,15 +436,15 @@ export default function ProjectPayoutMods({
           }}
           block
         >
-          <Trans>Add a split</Trans>
+          <Trans>Add a payout</Trans>
         </Button>
       </Space>
 
       <Modal
```

Comment:
The original `DistributionSplitModal`, which is now it's own file that handles everything itself (doesn't rely on `index` for anything other than finally setting the split)